woocommerce/woocommerce
# Code Freeze CLI Add changelog to code freeze action

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

#### Prerequisites
1. Fork woocommerce onto your own namespace. Push this branch, (`add/code-freeze-changelog-action`) to your fork

#### Test the action
1. Get `trunk` up to date on fork as well be working with the code freeze for 7.8
3. Make a 7.7.0 tagged release on fork
4. Delete `release/7.8` branch on fork if it already exists
5. Delete `prep/trunk-for-next-dev-cycle-7.9` on your fork if it already exists
6. Delete `update/7.8-changelog` on your fork if it already exists
7. Delete `delete/7.8-changelog` on your fork if it already exists
8. Delete 7.9.0 milestone if it already exists
9. Run the `Release: Code freeze` action from branch `add/code-freeze-changelog-action`. You'll want to add a time override so the action runs because its not code freeze day. `2023-04-17T15:00:00.000` works well.
10. Make sure the action passes.
11. Once the command has completed, you will see three pull requests. The changes in this PR only affect the changelog ones. For reference here are the two changelog pull requests from the 7.7 release. 
* https://github.com/woocommerce/woocommerce/pull/37776
* https://github.com/woocommerce/woocommerce/pull/37775
